### ISSUE TITLE
[OS X] Commander Can't move "._" files 

### ISSUE THREAD

**Issue by axifive** [trol73/mucommander#93]
I found a small bug. When I move dirs from usb flash to Hard Drive trolCommander(muCommander) show Dialog with message that it can't move all os x special `"._"` files. When original file/dir move special `"._"` file automaticaly delete. 
I think trolCommander(muCommander) must ignore all `"._"` moving files.

![2015-12-24 18 15 44](https://cloud.githubusercontent.com/assets/844963/11996269/7053ab66-aa70-11e5-9549-43ab239fe666.png)
![2015-12-24 18 16 07](https://cloud.githubusercontent.com/assets/844963/11995991/5f5b87d4-aa6b-11e5-8e64-0276007434a8.png)
![2015-12-24 18 17 16](https://cloud.githubusercontent.com/assets/844963/11995998/6da54c76-aa6b-11e5-930b-74656596f688.png)

**Comment by trol73** [comment]
Can't reproduce it. I can copy file with name "._"  or "._501" between folders on hard drive

**Comment by axifive** [comment]
Yes. on hard drive copy it maybe work, but I wrote about move these files from USB flash.
